src/scenes/AcceleratorRouter/Scoring/ScoreEntry.tsx

```diff
@@ -0,0 +1,148 @@
+import { useState } from 'react';
+
+import { Box, Button, Typography, useTheme } from '@mui/material';
+
+import TextArea from 'src/components/common/TextArea';
+import { useLocalization } from 'src/providers';
+import strings from 'src/strings';
+import { Score, ScoreValue, ScoreValues, getScoreCategory, getScoreColors, getScoreValue } from 'src/types/Score';
+
+type ScoringEntryProps = {
+  disabled?: boolean;
+  onChange: (value: ScoreValue) => void;
+  onChangeText: (id: string, value: unknown) => void;
+  phase: string;
+  qualitativeInfo: string;
+  score: Score;
+};
+
+const ScoreEntry = ({ disabled, onChange, onChangeText, phase, score, qualitativeInfo }: ScoringEntryProps) => {
+  const theme = useTheme();
+  const scoreCategory = getScoreCategory(score.category);
+  const { activeLocale } = useLocalization();
+
+  return activeLocale ? (
+    <Box sx={{ margin: '16px 0', width: 1 }}>
+      <Box sx={{ marginBottom: '8px' }}>
+        <Typography sx={{ color: theme.palette.TwClrBaseBlack, fontWeight: 500 }}>{phase}</Typography>
+        <Typography sx={{ color: theme.palette.TwClrBaseBlack, fontSize: '18px', fontWeight: 500 }}>
+          {scoreCategory} {strings.SCORE}
+        </Typography>
+      </Box>
+
+      <ScoreControl disabled={disabled} onChange={onChange} score={score} />
+
+      <Box>
+        <Typography sx={{ color: theme.palette.TwClrTxtSecondary, fontSize: '14px', marginBottom: '4px' }}>
+          {strings.QUALITATIVE_INFO}
+        </Typography>
+        {disabled ? (
+          <Typography sx={{ color: theme.palette.TwClrBaseBlack }}>{qualitativeInfo}</Typography>
+        ) : (
+          <TextArea id='qualitative-info' label='' value={qualitativeInfo} onChange={onChangeText} />
+        )}
+      </Box>
+    </Box>
+  ) : null;
+};
+
+export default ScoreEntry;
+
+type ScoreControlProps = {
+  disabled?: boolean;
+  onChange: (value: ScoreValue) => void;
+  score: Score;
+};
+
```

Comment:
There are two sub-components in this file: `SourceControl` and `SourceControlButton`. I can break these out into discrete files if preferred.